### DISCUSSION TITLE
docs: supported wallets are ambiguous?

### DISCUSSION THREAD

**Discussion by a3s7p** [NillionNetwork/discussions#127] [Ideas]
[It appears](https://docs.nillion.com/guide-testnet-connect#supported-wallets) that Leap wallet is supported but instructions are only provided for Keplr.

On the other hand, Metamask [can](https://verifier.nillion.com) be used but is not mentioned in the docs.

**Comment by wwwehr** [comment]
Thanks for the suggestion @a3s7p 

As you noted, we don't have official guidance around Leap wallet - but it is reported to work.

For our verifier project purpose - we activate both ETH (matamask) and NIL (keplr) wallets. I know cosmos chains can be added to metamask, but given that you are managing real value (ETH) we don't suggest experimental metamask usage (snaps).

I've captured the Leap wallet request and will close this discussion, but please reply if you have further questions.

**Comment by oceans404** [comment]
Added an Issue https://github.com/NillionNetwork/nillion-docs/issues/108
